src/main/java/com/coveo/spillway/storage/RedisStorage.java

```diff
@@ -92,7 +91,6 @@ public Map<LimitKey, Integer> addAndGet(Collection<AddAndGetRequest> requests) {
       try (Pipeline pipeline = jedis.pipelined()) {
 
         for (AddAndGetRequest request : requests) {
-          pipeline.multi();
```

Comment:
this is not transactional anymore though. Transactions are available on the `Jedis` instance and AFAICT, this is compatible on jedis 3.x and 4.x (not sure about 2.x) so it might keep the backward compatibility. 

I'm not familiar with why this stuff was transactional but I'm hoping it was for a good reason

https://github.com/redis/jedis/wiki/AdvancedUsage#transactions

Reply:
yep I changed that